### PR TITLE
Fix 5688

### PR DESCRIPTION
**Pull request by palango**
## Description

Fixes: #5688

This updates the sync check for the Raiden node, because etherscan will require an API key soon.

I chose to use https://api.blockcypher.com/v1/eth/main instead. That is only available for mainnet though, so the check is only done there and otherwise we rely on `eth_syncing`.
While implementing this I actually started to doubt that using an external API gives us a big advantage, so I'd also be fine with removing all that code and just use `eth_syncing` instead.

While testing I also found a minor error in a mainnet warning. This is fixed in the third commit.

### CODE REVIEW

**Comment by karlb** [comment]
Using an external API has the benefit of warning us when our eth node falls behind.

**Comment by palango** [comment]
> Using an external API has the benefit of warning us when our eth node falls behind.

Sure, but as long as the client isn't completely disconnected from the swarm, `eth_syncing` should do that as well.

File: raiden/ui/checks.py [hunk 1]
```diff
@@ -194,20 +194,4 @@ def check_pfs_configuration(pathfinding_service_address: str) -> None:
 
 
 def check_synced(rpc_client: JSONRPCClient) -> None:
```
Comment (karlb) [inline raiden/ui/checks.py]: We could just inline this function, now that only one line is left. It's only called in a single place.

**Comment by LefterisJP** [comment]
The reason we relied on external node is tha eth_syncing, at least 2 years ago was unreliable.

**Pull request merged**